pkg/templates/sample/report/ReportTesterTemplate.gen.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package istio.mixer.adapter.sample.report;
+
+import "mixer/v1/config/descriptor/value_type.proto";
+import "mixer/pkg/templates/mixer/TemplateExtensions.proto";
+
+option (istio.mixer.v1.config.template.template_variety) = TEMPLATE_VARIETY_REPORT;
+option (istio.mixer.v1.config.template.template_name) = "Sample";
+
+/*
+Data from Template proto
+message Constructor {
+    istio.mixer.v1.config.descriptor.ValueType value = 2;
+    map<string, istio.mixer.v1.config.descriptor.ValueType> dimensions = 3;
+}
+*/
```

Comment:
Do we need this as a comment anymore? I think it's been superseded by `Type` in this proto.